awesome_cart/awc.py

```diff
@@ -1087,6 +1089,13 @@ def cart(data=None, action=None):
 				quotation.shipping_address_name = address_name
 				awc_session["last_shipping_address"] = address_name
 
+		# check and update use_customer_fedex_account field in quotation
+		customer_fedex_acc = data[0].get("address").get("use_customer_fedex_account")
```

Comment:
Is it possible that data[0] may not be there? In that case this will break things

Reply:
nope not possible, it will always exist